src/api/register_request.rs

```diff
@@ -44,11 +42,16 @@ fn register_output(api: &mut Api, request: RegisterOutputRequest) -> Result<(),
     let RegisterOutputRequest {
         output_id,
         port,
-        resolution,
-        encoder_settings,
+        video,
         ip,
+        ..
     } = request;
 
+    let Some(video) = video else {
+        // TODO error
+        return Ok(());
```

Comment:
:smiling_face_with_three_hearts:

Reply:
Yep, it's just a spike and it's not meant to be merged anyway.
I don't want to spend time fixing implementation when it's not going to be used anyway.
This PR is mostly for API design